### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by azakus**
Update selenium-standalone for latest browser drivers
Update chalk

### CODE REVIEW

File: src/selenium.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@
  * Code distributed by Google as part of the polymer project is also
  * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
  */
-import * as chalk from 'chalk';
+import chalk from 'chalk';
```
Comment (justinfagnani) [inline src/selenium.ts]: `import chalk = require('chalk');`?

**Comment by paladox** [comment]
the selenium-standards update is being done in https://github.com/Polymer/wct-local/pull/71 :).

It includes other changes to fix this running on ci too.

**Comment by jaichandra** [comment]
Im running into console errors in Chrome headless. This probably will fix the issue. Waiting to be merged.

File: src/selenium.ts (hunk 1 above)
Comment (azakus) [reply src/selenium.ts]: sure

Comment (azakus) [reply src/selenium.ts]: Actually, this seems to break typing.

**Comment by paladox** [comment]
Needs rebase.

File: src/selenium.ts (hunk 1 above)
Comment (justinfagnani) [reply src/selenium.ts]: Huh, chalk must have change to module-compatible exports

**Comment by azakus** [comment]
@justinfagnani rebased with #71, with a few typing fixes for `browserOptions`.

**Pull request merged**